charts/orchestrator/templates/rhdh-operator.yaml

```diff
@@ -292,13 +292,13 @@ data:
                email:
                  transportConfig:
                    transport: smtp
-                   hostname: my-smtp-server
-                   port: 587
+                   hostname: {{ .Values.rhdhPlugins.notifications_email.hostname }}
```

Comment:
the entire plugin should be enabled/disabled based on the availability of the properties rather than setting dummy values.